src/general/WorldGenerationSettings.cs

```diff
@@ -130,6 +130,27 @@ public enum ClimateInstabilityEnum
         High = 2,
     }
 
+    /// <summary>
+    ///   The possible compound levels settings for the planet
+    /// </summary>
+    public enum CompoundLevelsEnum
```

Comment:
I think this should be just named `CompoundLevel` as the "enum" suffix in here in the others is used just as a way to avoid name conflicts, but this enum to me doesn't seem to have a name conflict so this should follow the usual naming of *not* having `Enum` suffix in the name.